cmd/ctl/pkg/status/certificate/certificate.go

```diff
@@ -150,21 +151,88 @@ func (o *Options) Run(args []string) error {
 	fmt.Fprintf(o.Out, fmt.Sprintf("Not After: %s\n", formatTimeString(crt.Status.NotAfter)))
 	fmt.Fprintf(o.Out, fmt.Sprintf("Renewal Time: %s\n", formatTimeString(crt.Status.RenewalTime)))
 
+	// TODO: What about timing issues? When I query condition it's not ready yet, but then looking for cr it's finished and deleted
+	// Try find the CertificateRequest that is owned by crt and has the correct revision
+	reqs, err := o.CMClient.CertmanagerV1alpha2().CertificateRequests(o.Namespace).List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return err
+	}
+	req, err := findMatchingCR(reqs, crt)
+	if err != nil {
+		return err
+	}
+	if req == nil {
+		fmt.Fprintf(o.Out, fmt.Sprintf("No CertificateRequest found for this Certificate\n"))
+	} else {
+		fmt.Fprintf(o.Out, fmt.Sprintf("CertificateRequest:%s\n", crInfoString(req)))
```

Comment:
I wonder if it would be useful to have a cli flag to show the CR history. I don't know that it is, just if you find
that it would be useful when trying this out or seeing other people use it then we could add it.